readthedocs/builds/migrations/0054_add_builds_index_for_addons.py

```diff
@@ -13,7 +13,12 @@ class Migration(migrations.Migration):
         migrations.AlterIndexTogether(
             name="build",
             index_together={
-                ("date", "id"),
+                # TODO: something changed in Django 5.2 that I need to comment this index together here.
+                # We need to research a little more.
+                #
+                # ValueError: Found wrong number (0) of constraints for builds_build(date, id)
+                #
+                # ("date", "id"),
```

Comment:
Same here. I had to delete this chunk of code to be able to run the tests. I don't think this will have any impact, tho.